LL/h_ceinselaig.trig

```diff
@@ -168,10 +167,14 @@
     irishRel:nomName "Enna Cennselach";
     rel:childOf <#Labrada>;
     owl:sameAs <http://example.com/LL/genelach_h_cendselaig.trig#EnnaGensalach>.
-    
-# sameAs http://example.com/LL/dal_corpri_arad.trig#Chendselaig ? - CY
-# Yup, but I found a more direct mention in that tract, hiding beneath 
-# a weird spelling variant. EPT
+
+<< <#EnnaeCendselaig>
+        rdfs:comment "sameAs http://example.com/LL/dal_corpri_arad.trig#Chendselaig ?" >>
+        prov:wasAttributedTo <https://orcid.org/0000-0002-7241-3264> .
+
+<< <#EnnaeCendselaig>
+        rdfs:comment "Yup, but I found a more direct mention in that tract, hiding beneath a weird spelling variant." >>
+        prov:wasAttributedTo <https://orcid.org/0000-0003-2819-5519> .
```

Comment:
Is this an instance where nesting could be used to show the sequence in which the comments were written?